backend/census_historical_migration/sac_general_lib/sac_creator.py

```diff
@@ -81,6 +81,14 @@ def setup_sac(user, audit_header):
     sac.auditee_certification = auditee_certification(audit_header)
     sac.auditor_certification = auditor_certification(audit_header)
     sac.data_source = settings.CENSUS_DATA_SOURCE
+
+    if audit_header.ENTITY_TYPE == "Tribal":

```

Comment:
```suggestion
    if general_info["entity_type"] == "tribal":

```

Reply:
That was a typo, `entity_type` is the name in dissemination use `general_info["user_provided_organization_type"]` instead.

Reply:
Done